packages/web-react/src/components/CheckboxField/CheckboxField.stories.ts

```diff
@@ -14,6 +14,9 @@ export default {
 } as ComponentMeta<typeof CheckboxField>;
 
 export { default as CheckboxField } from './stories/CheckboxField';
-export { default as CheckboxFieldError } from './stories/CheckboxFieldError';
+export { default as CheckboxFieldHelperText } from './stories/CheckboxFieldHelperText';
+export { default as CheckboxFieldDanger } from './stories/CheckboxFieldDanger';
```

Comment:
First look and I suggest creating CheckboxFieldValidationState story where all three states will be. Same as we have it with colors.

Reply:
done